File: tests/hostile/hello.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"sort"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	line, _ := reader.ReadString('\n')
	fmt.Println("hello from go")
	fmt.Println(strings.ToUpper(strings.TrimSpace(line)))
	v := []int{3, 1, 2}
	sort.Ints(v)
	fmt.Println(v)
}

File: tests/hostile/hello.rs
use std::io::Read;

fn main() {
    let mut input = String::new();
    std::io::stdin().read_to_string(&mut input).ok();
    println!("hello from rust");
    println!("{}", input.trim().to_uppercase());
    let mut v = vec![3, 1, 2];
    v.sort();
    println!("{:?}", v);
}
